### PR TITLE
feat: add position option

### PR DESCRIPTION
**Pull request by hyoban**
This helps us customize the style

### CODE REVIEW

**Comment by arjunvegda** [comment]
Hey @hyoban 

Thanks for the PR! 🙌 Does overriding `.jotai-devtools-trigger-button` in your application not work?  We expose `jotai-devtools-shell` too, if you'd like to add minor styles to the shell.

**Comment by hyoban** [comment]
> Hey @hyoban 
> 
> 
> 
> Thanks for the PR! 🙌 Does overriding `.jotai-devtools-trigger-button` in your application not work?  We expose `jotai-devtools-shell` too, if you'd like to add minor styles to the shell.

Yes, I'm using this method now, it works. But with tailwind, it will be more convenient if you can pass the classname.

**Comment by arjunvegda** [comment]
> > Hey @hyoban
> > Thanks for the PR! 🙌 Does overriding `.jotai-devtools-trigger-button` in your application not work?  We expose `jotai-devtools-shell` too, if you'd like to add minor styles to the shell.
> 
> Yes, I'm using this method now, it works. But with tailwind, it will be more convenient if you can pass the classname.

What sort of CSS overrides do you need? Trying to understand the use case better. We'd want to limit the customizability via new class names. Also, I am not sure if supplying the same `className` for Trigger and Shell is a good idea.

**Comment by hyoban** [comment]
> What sort of CSS overrides do you need? Trying to understand the use case better. 

The main thing is the location and size. Maybe it would be more convenient to drag and zoom as we need.

> We'd want to limit the customizability via new class names. Also, I am not sure if supplying the same `className` for Trigger and Shell is a good idea.

Yeah I just realized that. Provide two different `classname`?

**Comment by arjunvegda** [comment]
> The main thing is the location and size. Maybe it would be more convenient to drag and zoom as we need.

Got it! I think, for now, we'd want to limit overriding the class names. Could you please work with the ones that we provide out of the box instead of accepting them as props? 

For the current PR, could you please update it to only include `position` logic? That sounds like something we'd love to support out of the box. And of course, happy to accept PRs for the drag/drop feature too.

File: src/DevTools/DevTools.tsx [hunk 1]
```diff
@@ -83,10 +58,49 @@ const DevToolsMain = ({
...
+          width: '4rem',
+          height: '4rem',
+          zIndex: 99999,
+          img: {
+            height: '2rem',
+          },
+          left: position.includes('left') ? '0.2rem' : 'unset',
+          right: position.includes('right') ? '0.2rem' : 'unset',
+          top: position.includes('top') ? '0.2rem' : 'unset',
+          bottom: position.includes('bottom') ? '0.2rem' : 'unset',
```
Comment (arjunvegda) [inline src/DevTools/DevTools.tsx:98]: Could you add some tests for this? Could be as simple as checking for these positions based on the prop

**Comment by arjunvegda** [comment]
Could you also update the `readme.md` file with this prop?

**Comment by hyoban** [comment]
@arjunvegda Updated

**Review by arjunvegda — APPROVED** [review]

**Comment by arjunvegda** [comment]
I'll release this shortly! Thanks again for your contribution!

**Comment by arjunvegda** [comment]
Thanks for your patience, this change has been released as part of `0.8.0` 🎉

**Pull request merged**